sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_service_client_async.py

```diff
@@ -465,9 +465,8 @@ def get_container_client(self, container):
         The container need not already exist.
 
         :param container:
-            The container. This can either be the name of the container,
-            or an instance of ContainerProperties.
-        :type container: str or ~azure.storage.blob.models.ContainerProperties
+            The container that the blob is in.
+        :type container: Union[ContainerProperties, str]
```

Comment:
:type container: str or ~azure.storage.blob.ContainerProperties

The Union syntax is for typing and is not recognized by sphinx

Reply:
fixed